app-admin/kubelet/kubelet-1.1.2-r2.ebuild

```diff
@@ -11,8 +11,9 @@ inherit flag-o-matic systemd toolchain-funcs
 DESCRIPTION="Kubernetes Container Manager"
 HOMEPAGE="http://kubernetes.io/"
 KEYWORDS="amd64"
-SRC_URI="https://github.com/GoogleCloudPlatform/kubernetes/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-S="${WORKDIR}/kubernetes-${PV}"
+MY_PV="${PV/_p/_coreos.}"
+SRC_URI="https://github.com/coreos/kubernetes/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
No, this must use MY_P for the final file name, as is it is conflicting with the previous file name.